addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -191,6 +237,43 @@ function ComposerFactory({ Entity }) {
 
         /**
          *
+         * Generates the html link related to the mentionned partner
+         *
+         * @private
+         * @param {string} body
+         * @returns {string}
+         */
+        _generateMentionsLinks(body) {
```

Comment:
It would be best if we could find a way to use templates instead of having to hardcode html like here...
To be discussed with @alexkuhn, maybe out of scope.

Reply:
I'm good with hardcode html for the moment. Templates introduce more issues at the moment, so I wouldn't bother with this.